### ISSUE TITLE
Misleading example

### ISSUE THREAD

**Issue by peske** [lesaux/puppet-kibana4#69]
I've installed Kibana 4.5.1 by using this module, and as I can see it already has Sense embedded. If so please remove the following part from your example (README.md file):

```
'elastic/sense' => {
  ensure          => present,
  plugin_dest_dir => 'sense',
},
```

It caused me a headache and wasting few hours trying to install Sense (as you can see [here](https://github.com/elastic/kibana/issues/7400)) before I've realized that it is already installed :)

**Comment by peske** [comment]
It looks that I'm wrong. I'll be back soon with the explanation.

**Comment by peske** [comment]
Well yes, definitely my mistake. I'm not sure what I've done wrong earlier, but someway I've managed to install Sense who knows where, and kibana4 module was trying again and again to install it because 'sense' directory wasn't in kibana/installedPlugins. 

I'm still not sure how it happened, but the fact is that I've installed Sense by using your plugin (and the example code), yet obviously it was installed who knows where. Maybe because I've installed Marvel and Sense in the same agent run? Who knows...

Anyway, after reinstalling everything (except for Marvel), I have Sense installed in installedPlugins directory.
